src/fonts/Font.ts

```diff
@@ -261,21 +262,19 @@ export class Font {
         new Vec2(width * scale + padding * 2, height * scale + padding * 2)
       );
 
-      positionX += ((i !== 0 ? lsb : 0) + width + rsb) * scale;
+      positionX += (lsb + width + rsb) * scale;
     }
 
-    // Skip the last padding.
-    const width =
-      positions[positions.length - 1].x + sizes[sizes.length - 1].x - padding;
+    const width = positions[positions.length - 1].x + sizes[sizes.length - 1].x;
     const height =
       (this.metadata.capHeight * fontSize) / this.metadata.unitsPerEm;
 
     return {
       positions,
       sizes,
       boundingRectangle: {
-        width: Math.ceil(width),
-        height: Math.ceil(height),
+        width,
+        height,
```

Comment:
Rounding sizes is handled in the renderer so this can be removed to prevent inaccuracies from double rounding.